basyx/aas/adapter/http.py

```diff
@@ -297,18 +299,18 @@ def json_list(cls, data: Union[str, bytes], expect_type: Type[T], stripped: bool
         return [cls.assert_type(obj, expect_type) for obj in parsed]
 
     @classmethod
-    def base64json_list(cls, data: Union[str, bytes], expect_type: Type[T], stripped: bool, expect_single: bool)\
+    def base64urljson_list(cls, data: Union[str, bytes], expect_type: Type[T], stripped: bool, expect_single: bool)\
```

Comment:
base64 encoded data is always represented as a string, thus the type of `data` should be changed to `str`. This should also help resolve the remaining mypy errors. Same for the `base64urljson` function.